### DISCUSSION TITLE
Plot stop using SSD

### DISCUSSION THREAD

**Discussion by Simfiz** [Chia-Network/chia-blockchain#4737] [Q&A]
Hi guys, have a problem with parallel ploting
While creating 2 plots in parallel mode, one of them stop use SSD and working at all, but it use other resources (CPU and RAM).
It happens not always, but often.
During the plotting nothing except chia and hpool is running.
I don't get any errors in debug log and plotting log.
https://drive.google.com/file/d/1_dG5Z_e0BU7KipGk33cqJ64ArSobJz5j/view
https://drive.google.com/file/d/1ppWvidwa1QwgH0ZeUmUXGRAPVFcJLzNd/view?usp=sharing
https://drive.google.com/file/d/1GXAgKgPB_2WA21iZ8tcYFmuuWAgVi-JQ/view?usp=sharing
During time its still no working at all.

my HW:
R5 3600
16gb RAM 2*8 3600
1 TB 980 Samsung nvme m2
HDD - new IronWolf 6TB
Fresh win 10 - i reinstall it 2 days ago.

Plotting setings:
2 in parallel
40 min delay
12 threads
3900 or 4000 RAM
Temp files on Samsung 980
Local files on HDD

**Comment by Spider4896** [comment]
Your telling the system to assign 12 threads to each plot and your system only has 12.
Try using just 2 threads.
